net2/FlowTool.js

```diff
@@ -573,17 +539,7 @@ class FlowTool {
       if (x.ets) x.ts = x.ets
     });
 
-    let mergedFlow = null
-
-    if(!options.no_merge) {
-      mergedFlow = this._mergeFlows(
-        _.orderBy(flowObjects, 'ts', options.asc ? 'asc' : 'desc')
-      );
-    } else {
-      mergedFlow = flowObjects
-    }
-
-    let simpleFlows = mergedFlow
+    let simpleFlows = flowObjects
```

Comment:
There's still reference to this function in TopTransfer.js while `last60stats` is not actually used any more. Remove those code/file to avoid confusion.